src/components/nowplayingbar/nowplayingbar.js

```diff
@@ -281,22 +279,13 @@ define(['require', 'datetime', 'itemHelper', 'events', 'browser', 'imageLoader',
     }
 
     function updatePlayPauseState(isPaused) {
-        var i;
-        var length;
-
         if (playPauseButtons) {
-            if (isPaused) {
-
-                for (i = 0, length = playPauseButtons.length; i < length; i++) {
-                    playPauseButtons[i].querySelector('i').innerHTML = '&#xE037;';
-                }
-
-            } else {
+            let icons = ['play_arrow', 'pause'];
+            if (isPaused) icons = icons.reverse();
```

Comment:
```suggestion
            let icons = isPaused ? ['pause', 'play_arrow'] : ['play_arrow', 'pause'];
```
maybe this is easier to understand?

Reply:
At first I didn't use an array. Like here https://github.com/jellyfin/jellyfin-web/blob/d98645135b53612d852bf14991d247880f502265/src/components/remotecontrol/remotecontrol.js#L352-L358
Then I decided that icons are repeated a lot and packed them into an array.

More forgivable way is "remove+add": https://github.com/jellyfin/jellyfin-web/pull/1131#discussion_r418436749
In this case, you do not need to set initial icon (which will be the target for `replace`).